modules/upgrade/pages/db-migration-13.adoc

```diff
@@ -0,0 +1,124 @@
+[[db-migration-13]]
+= Database Migration from Version 12 to 13
+
+This section covers upgrading the PostgreSQL database from version{nbsp}12 to version{nbsp}13.
+If you are already using PostgreSQL 13, you do not need to perform this migration.
+If you are using version 10, see xref:upgrade:db-migration-12.adoc[].
+If you are using an older version, such as version 9.6, see xref:upgrade:db-migration-10.adoc[].
+
+If you want to upgrade to the latest {productname} version, you must be using PostgreSQL version 12 or 13, depending on the underlying operating system. When running on SLES 15 SP3, version 13 should be used. When running on Leap 15.2, version 12 should be used.
+
+
+
+////
+== Background Information
+
+
+* {productname} 3.1.4, SLES 12 SP3, Postgres 9.6
+* {productname} 3.1.11, SLES 12 SP3, Postgres 9.6
+* {productname} 3.2.10, SLES 12 SP3, Postgres 9.6
+* {productname} 3.2.10, SLES 12 SP3, Postgres 10
+* {productname} 3.2.10, SLES 12 SP4, Postgres 10
+* {productname} 4.0.0, SLES 15 SP1, Postgres 10
+* {productname} 4.1.0, SLES 15 SP2, Postgres 12
+* {productname} 4.2.0, SLES 15 SP3, Postgres 13
+
+////
+
+
+
+
+[[db-migration-13-prepare]]
+== Prepare to Upgrade
+
+Before you begin the upgrade, prepare your existing {productname} Server and create a database backup.
+
+PostgreSQL stores data at [path]``/var/lib/pgsql/data/``.
+
+.Procedure: Preparing to Upgrade
+
+. Check the active PostgreSQL version:
++
+----
+psql --version
+----
++
+If you are using PostgreSQL{nbsp}12, you can upgrade to PostgreSQL{nbsp}13.
++
+If you are already using PostgreSQL version 13, you do not need to perform this migration.
+. Check the active smdba version:
++
+----
+rpm -q smdba
+----
++
+// CHECKIT!
++
+PostgreSQL{nbsp}13 requires ``smdba`` version 1.7.6 or later.
+
+. Perform a database backup.
+    For more information on backing up, see xref:administration:backup-restore.adoc[].
+
```

Comment:
Needs some whitespace adjustments, but should render OK.

Reply:
which whitespaces do you refer to?

Reply:
No need to put newlines between steps, and we usually put three newlines before a title. But it's just cosmetic, so no big deal.

Reply:
jejeje, what is "steps" and what is "title" ?

Reply:
Let's just merge this as-is, and I'll fix the table in a new PR.